### ISSUE TITLE
"RECONNECT" button text is not visible in High Contrast Black Mode.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#370]
### Steps to Reproduce

1. Change the High contrast standards to High Contrast black before opening the application.
2. Launch Xamarin Workbooks application and create a new workbook with IOS platform.
3. Observe "RECONNECT " button text at the footer.

### Expected Behavior
Should be visible in High contrast black mode
### Actual Behavior
"RECONNECT" button text is not visible in High Contrast Black Mode.
Note: In High contrast white "Reconnect" text is clearly visible (Refer screenshot)
### User Impact
User cannot see the "RECONNECT" button.
### Screenshots
![17 hcb](https://user-images.githubusercontent.com/22956130/40725666-a4594ed8-6441-11e8-869d-3c251a8da2e3.png)
![17 hcw](https://user-images.githubusercontent.com/22956130/40725667-a4919d7e-6441-11e8-968d-c47f379507ef.png)
![17 normal](https://user-images.githubusercontent.com/22956130/40725668-a4c87a74-6441-11e8-8a0c-92ee4e6906ad.png)

### Environment
Windows : Version 1803 (OS Build : 17134.81)
Xamarin Version : 1.4(master/b3f92f9)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS4.3.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
